src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const list = document.querySelector('#thumbs');
+
+list.onclick = function (item) {
```

Comment:
Consider checking if the `target` is an image element before accessing `target.src`. This can prevent errors if the clicked element is not an image.